.eslintrc

```diff
@@ -24,7 +24,10 @@
       "error",
       "always"
     ],
-    "comma-dangle": ["error", "never"],
+    "comma-dangle": [
```

Comment:
out of curiosity - why this change? The array is too short for multiline values

Reply:
I think my editor probably formatted it like this after saving this file @sachungo